conduct.md

```diff
@@ -40,25 +40,34 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Be proactive about communicating.
+
+- Constructively criticize others.
+
+- Acknowledge others.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are discouraged to:
+
+- Be passive aggressive/disrespectful.
+
+- Never ask questions.
+
+- Never listen to criticism/feedback.
 
 ### Participation
 
-TODO: Write an introductory paragraph.
+Students should participate in a meaningful capacity during this course. This section will outline the acceptable and unacceptable conduct when it comes to class participation. Students should follow these guidelines when participating in classroom activities.
```

Comment:
It would be useful to have more definitive language for this paragraph -- 'Students are expected to' rather than 'Students should'